Makefile

```diff
@@ -0,0 +1,4 @@
+test:
+	docker-compose up -d
+	cargo test -- --test-threads 1
```

Comment:
Har ikke akkurat full kontroll her, men er det noen av testene som kjøres som faktisk trenger at docker-compose har startet?

Alt så for min del ut som enhetstester

Reply:
Så input_message i TestProducer blir sendt til kafka? :+1:

Reply:
Stemmer!